### PR TITLE
Update links to Azure samples

### LINKED ISSUE #16257: 404 broken link on documentation site
Broken link to https://azure.microsoft.com/documentation/samples/active-directory-dotnet-webapp-openidconnect-aspnetcore/
From the menu, this seems to have been already reported in #15139

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #16257

Thanks @freemstr! 🎷 Sorry for the misunderstanding. I searched the docs markdown files (*.md*) thinking it was in a topic, but our TOC (table of contents) file for the sidebar TOC is actually a YAML file (*.yml*), so I missed it. *Thanks for your persistence!*

### CODE REVIEW

File: aspnetcore/toc.yml [hunk 1]
```diff
@@ -883,15 +883,15 @@
             - name: Overview
               uid: security/authentication/azure-active-directory/index
             - name: Integrate Azure AD into a web app
-              href: https://azure.microsoft.com/documentation/samples/active-directory-dotnet-webapp-openidconnect-aspnetcore/
+              href: https://github.com/Azure-Samples/active-directory-dotnet-webapp-openidconnect-aspnetcore
```
Comment (scottaddie) [inline aspnetcore/toc.yml]: This link has been archived. Consider replacing with the recommended link (https://github.com/Azure-Samples/ms-identity-aspnetcore-webapp-tutorial).

**Comment by guardrex** [comment]
Stand-by ... have one more link to update in the topic itself.

**Comment by guardrex** [comment]
Thanks for the catch ... I guess u saw that in the banner. I was just happy the darn thing didn't 404! 🙈

**Comment by guardrex** [comment]
**_OH!_** The text of the README. Gotcha. Good :eye:!

**Comment by scottaddie** [comment]
Yes, the README file contained the new link.

**Pull request merged**